e2e/setup-kops.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
This file isn't strictly necessary, but included for homogeneity with setup-kube.sh